src/components/ha-sidebar.html

```diff
@@ -197,6 +201,10 @@
     this._boundUpdateStyles = this.updateStyles.bind(this);
   },
 
+  tr: function (pg, text, options) {
```

Comment:
That's good with me, with the following caveats:

1) It looks to me that with the AppLocalizeBehavior, each element needs to load it's resources separately. I think for the hass use case it definitely makes more sense to load all of the app resources in one request, since that's how the rest of the app is structured.

2) The dependency looks quite a bit larger, and maybe is overkill: https://github.com/yahoo/intl-messageformat/blob/master/dist/intl-messageformat.min.js (Still not huge, but just a consideration)